### ISSUE TITLE
[Docs Feedback]: Make "Install JetBrains Gateway instructions" more failsafe

### ISSUE THREAD

**Issue by geropl** [gitpod-io/website#2890]
### Page URL

https://www.gitpod.io/docs/integrations/jetbrains-gateway#getting-started

### User feedback

When telling users to install JetBrains gateway, we send them off to an external document hosted by JetBrains [here](https://github.com/gitpod-io/website/blob/main/gitpod/docs/integrations/jetbrains-gateway.md?plain=1#L36). After an additional jump (!) one lands on a document that proposes three "[installation scenarios](https://www.jetbrains.com/help/idea/jetbrains-gateway.html)" (?):
 1. "Open JetBrains Gateway from your local IntelliJ IDEA"
 2. "Install JetBrains Gateway via ToolBox"
 3. "Install JetBrains Gateway manually﻿"

On Monday (10.10.2022) some (MacOS) users following those instructions ended up in a dead end because of version incompatibilities between their locally installed JetBrains version (`stable`) and the "Gitpod plugin". After further experimenting it turned out that one needs to pick "scenario" 3 to make Gitpod work, because then the exact version of the local IDEA does not matter.
The process + documentation felt the very overwhelming and was effectively broken So even if this was a one-off situation (which it does not feel like, please correct me if I'm wrong here) the process felt like the opposite of frictionless.

Ideas to improve:
 1. Cut out the 1 additional jump to [this generic doc](https://www.jetbrains.com/help/idea/remote-development-a.html#gateway) (link is (https://github.com/gitpod-io/website/blob/main/gitpod/docs/integrations/jetbrains-gateway.md?plain=1#L36).
 1. Can we link to the manual instructions ("installation scenario 3") directly?
 2. Alternatively: Would it make sense to include those instructions directly? (maintenance!)

**Comment by loujaybee** [comment]
Thanks for the feedback @geropl ! With regards to Gateway download, longer term we'd like something like the ability to embed the gateway download into Gitpod so that users don't have to bounce to JetBrains docs and back to Gitpod 🙏 : 

- https://github.com/gitpod-io/gitpod/issues/8572

> ended up in a dead end because of version incompatibilities between their locally installed JetBrains version (stable) and the "Gitpod plugin". 

This is a slightly complex topic as we have to sync multiple versions: 

1. JetBrains deployed version (server)
2. JetBrains Gateway version
3. JetBrains plugin version (client)

I think we already have quite a few issues on this topic (needs pulling together in an epic most likely) and for sure it's something we need to improve, but we're also always iterating on the onboarding flow, so 100% we'll be coming back to this! We've raised a few issues with JetBrains, as some aspects do sit with them, and some with Gitpod. 🙏 

- https://github.com/gitpod-io/gitpod/issues/13654
- https://github.com/gitpod-io/gitpod/issues/12625
- https://github.com/gitpod-io/gitpod/issues/12141
- https://github.com/gitpod-io/gitpod/issues/11801

> The process + documentation felt very overwhelming and was effectively broken.

Do you have more feedback other than what you mentioned above, @geropl? I can try and speak to you sometime just to get a bit more understanding of what you're referring to and we can see what changes we can make! 🙏

**Comment by loujaybee** [comment]
> So even if this was an one-off situation (which it does not feel like, please correct me if I'm wrong here)

One aspect of this feedback that I'm not sure of is that why you needed to do a manual download to get this to work, this process should work just fine with toolbox, we need to take a look into that.

**Comment by geropl** [comment]
>>  The process + documentation felt very overwhelming and was effectively broken.
>
> Do you have more feedback other than what you mentioned above, @geropl? I can try and speak to you sometime just to get a bit more understanding of what you're referring to and we can see what changes we can make! pray

Happy to sync! But to maybe a little upfront: I was (almost) exclusively referring to the JetBrains documentation, in the context of "1st time Gitpod user wants to get their IDE working".
The judgment I present(ed) here is justified with my watching users going through these steps and me interpreting their reactions.

To expand a little:
 1. When you are a first time Gitpod user, our documentation introduces new JetBrains terminology ("JetBrains Gateway"). This is already tough, but well-managed by the "follow step 1-4 and you'll be fine" + video support.
 2. Things start to get tough when we send them off-site to another explanation page about what this "Gateway" is, where they have to find the link to "installation scenarios", and identify that as a link that get them closer to their goal: Get Gitpod working.
 3. Next user's have to at least skim this whole page to a) learn about new terminology ("installation scenario") and b) they have too choose one. I'd bet (speculation!) that most people don't understand the tradeoffs of the various scenarios, and "just want it to work" (TM).
 4. The installation method users choose in this situation (1) lead them into a dead end because of some version mismatch. (that's what I was referring to as "broken"). :+1:

**Comment by felladrin** [comment]
I agree that it would be easier if we showed only one way for users to install the gateway.

Asking users to install JetBrains Toolbox, enable the option "automatically update the tools" and then install Gateway through it would solve most of the problems related to Gateway.
